submit-web/src/routes/_authenticated/_dashboard/projects/index.tsx

```diff
@@ -1,10 +1,236 @@
+            }}
+          >
+            <Box
+              display={"flex"}
+              justifyContent={"space-between"}
+              sx={{ py: BCDesignTokens.layoutPaddingXlarge }}
+            >
+              <CardInnerBox>
+                <Typography variant="h4" fontWeight={400}>
+                  Management Plans
+                </Typography>
+                <ProjectStatus bold status={PROJECT_STATUS.POST_DECISION} />
+              </CardInnerBox>
+              <CardInnerBox>
+                <Button>
+                  <AddIcon sx={{ p: 0, mr: 0.5 }} />
+                  New Submission
+                </Button>
+              </CardInnerBox>
+            </Box>
+            <Box height={"100%"} px={BCDesignTokens.layoutPaddingXsmall}>
+              <Divider sx={{ mb: 0.5 }} />
+              <Typography
+                variant="body1"
+                sx={{
+                  fontWeight: "bold",
+                  backgroundColor: BCDesignTokens.themeGold10,
+                }}
+              >
+                Active Submissions
+              </Typography>
+              <CardInnerBox sx={{ height: "100%" }}>
+                <TableContainer component={Box} sx={{ height: "100%" }}>
+                  <Table
+                    width={"100%"}
+                    sx={{ tableLayout: "fixed" }}
+                    aria-label="simple table"
+                    stickyHeader
+                  >
+                    <TableHead>
+                      <TableRow>
+                        <TableCell sx={{ color: BCDesignTokens.themeGray70 }}>
+                          Submission Name
+                        </TableCell>
+                        <TableCell
+                          align="center"
+                          sx={{ color: BCDesignTokens.themeGray70 }}
+                        >
+                          Submitted On
+                        </TableCell>
+                        <TableCell
+                          align="center"
+                          sx={{ color: BCDesignTokens.themeGray70 }}
+                        >
+                          Submitted By
+                        </TableCell>
+                        <TableCell
+                          align="center"
+                          sx={{ color: BCDesignTokens.themeGray70 }}
+                        >
+                          Status
+                        </TableCell>
+                      </TableRow>
+                    </TableHead>
+                    <SubmissionTable plans={plans} />
+                  </Table>
+                </TableContainer>
+              </CardInnerBox>
+              <Typography
+                variant="body1"
+                sx={{
+                  fontWeight: "bold",
+                  backgroundColor: BCDesignTokens.themeGold10,
+                }}
+              >
+                Past Submissions
+              </Typography>
+              <CardInnerBox sx={{ height: "100%" }}>
```

Comment:
just take out everything under Active submissions into a separate component and everything under Past Submissions into a separate component